### PR TITLE
Ruby Catalog of things

### PR DESCRIPTION
**Pull request by Bria222**
## In this PR we Implemented the below:

- Created `main` file to start the app.
- Created `app` file that contains most of the project logic.
- Each one of us created a specific part of the app as follows:
               1. @AsimKhan2019  created the book class and module and all related methods tests
               2. @Fadahunsiseyi  album class and module and all related methods  and tests
               3. I created the games class and module and all related methods and tests
- Each one of us added unit tests to the classes he created.
- We implemented startup actions that allow the user to choose options and add data.
- Each of us preserved the incoming data into a related `json` file.
- Each of us added database tables analogical to the structure of the classes he created.

Here you can find our video presentation for the project:
[video link](https://drive.google.com/file/d/1BNQRHDFyamsiGgeciE6LUM0vN7wZhagi/view?usp=sharing)

Thank you 🎉🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+
+- GitHub: [@githubhandle](https://github.com/Fadahunsiseyi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fadahunsiseyi/)
+
+
+👤 **Brian Nyachae**
+
+- GitHub: [@githubhandle](https://github.com/Bria222)
+- Twitter: [@twitterhandle](https://twitter.com/briannyachae9)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/brian-nyachae/)
```
Comment (alexander16108) [inline README.md:59]: the following requirement should be met for your project to be finally approved 

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds attractive or is proud of making.

currently, your video recording does not have all the students present and has no audio with the length of the video exceeding its requirement of 5mins 😄

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
doing this is will be a simple task which will not require a lot of time to work on so congrats in advance and happy coding 

 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Bria222** [comment]
hi @alexander16108  thanks for the review we have worked on implimenting the amendments

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Bria222, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the main branch: **merged their feature branches into the dev branch**) 👏

 ## Required Changes ♻️

- [x] Please, as stated in the previous review, the maximum length of the video is 5 mn. Kindly record again. 
- [x] I would suggest that only one person should share the screen and all of you in the video will talk about the project. This will save time (not more than 5 minutes)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
Kindly please approve this pull request, my daughter is very sick, and I have to take her to the hospital right now for more tests and examinations. I am quite stressed out and unable to make the video every time. I might even have to spend the night at the hospital if the doctor advises for admission.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**